cf-usb/cf-usb.yaml

```diff
@@ -61,7 +86,30 @@ resources:
     secret_access_key: ((s3-secret-key))
     endpoint: ((s3-endpoint))
     disable_ssl: ((s3-disable-ssl))
-    regexp: ((s3-prefix))cf-usb-(.*).tgz
+    regexp: ((s3-prefix))cf-usb-sidecar-mysql-(.*).tgz
+- name: docker.postgres
+  type: docker-image
+  source:
+    repository: ((docker-server))((docker-org))/cf-usb-sidecar-postgres
+    tag: ((docker-tag))
+    username: ((docker-username))
+    password: ((docker-password))
+- name: docker.postgres-setup
```

Comment:
I think this means we're missing the `-db` image here too?

Reply:
Yes. I did not do the rebase needed yet yesterday, which is where I planned to put that in too.

Reply:
Done with commit 6638b9be7771e606e735e3be7778fe417d35d38d
Rebased too, to master.